### PR TITLE
Asset: Prevent duplicate files for being added in a group.

### PR DESCRIPTION
**Pull request by alejandroiglesias**
The title pretty much says everything. I believe this is very helpful.

### CODE REVIEW

**Comment by frankdejonge** [comment]
This is why you should always test your code before sending a pull request. Arr::path doesn't exist.

**Comment by alejandroiglesias** [comment]
I'm sorry, you're right. In fact I was developing in Kohana and used Arr::path(), but the wonderful FuelPHP already has dot-notated paths for arrays in Arr::get() ! I'm still in the transition from Kohana to FuelPHP and learning the simmilarities and differences.
I just commited the fix. Best regards.

**Comment by tomschlick** [comment]
Also I believe as per the fuel coding standards the "continue;" statement should be enclosed within braces.

http://fueldevdocs.exite.eu/general/coding_standards.html#/control_structures

It's not entirely explicit on not using them for one line statements but just from previous experience I believe so.

**Comment by jschreuder** [comment]
What @tomschlick said: all control structures should always use braces, no exceptions.

**Comment by alejandroiglesias** [comment]
Sorry about that, thanks to all for instructions. Already fixed that also.

File: classes/asset/instance.php [hunk 1]
```diff
@@ -455,6 +455,12 @@ protected function _parse_assets($type, $assets, $attr, $group)
 
 		foreach ($assets as $key => $asset)
 		{
+			// Prevent duplicate files in a group.
+			if (Arr::get($this->_groups, "$group.$key.file") == $asset)
```
Comment (frankdejonge) [inline classes/asset/instance.php]: Arr should be prefixed with a backslash to get it from global

**Comment by alejandroiglesias** [comment]
@FrenkyNet thanks. Fixed.

**Pull request merged**